js/debugger.ts

```diff
@@ -0,0 +1,59 @@
+import { debug, toHex } from './util';
+import { word } from './types';
+
+import SYMBOLS from './symbols';
+import CPU6502, { DebugInfo } from './cpu6502';
+
+const alwaysBreak = (_info: DebugInfo) => { return true; };
```

Comment:
Seeing that `(info: DebugInfo) => boolean` is used a bunch, maybe there should be a type for it?